spec/excon/rails_spec.rb

```diff
@@ -34,6 +34,14 @@ module Excon
         .to match(%r{Excon Request \(\d+\.\d+ms\)  GET http://example\.com/})
     end
 
+    it 'logs query params to logger' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.